dev/ci/report-bundle-diff.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "--- Find a commit to compare the bundle size against"
+# For testing purposes we assume that this is the latest main build
+# since we have no data for main yet.
+MERGE_BASE=$(git merge-base HEAD origin/main)
+export MERGE_BASE
+
+# We may not have a stats.json file for the merge base commit as these are only
+# created for commits that touch frontend files. Instead, we scan for 20 commits
+# before the merge base and use the latest stats.json file we find.
+REVISIONS=()
+while IFS='' read -r line; do REVISIONS+=("$line"); done < <(git --no-pager log "${MERGE_BASE}" --pretty=format:"%H" -n 20)
+for REVISION in "${REVISIONS[@]}"; do
+  gsutil -q cp -r "gs://sourcegraph_buildkite_cache/sourcegraph/sourcegraph/bundle_size_cache-$REVISION.tar.gz" "./ui/assets/bundle_size_cache-$REVISION.tar.gz" || true
+  tar -xvf "ui/assets/bundle_size_cache-${REVISION}.tar.gz" ui/assets || true
```

Comment:
```suggestion
  tar -xf "ui/assets/bundle_size_cache-${REVISION}.tar.gz" ui/assets || true
```

Removing `-v` will shave a few lines from the output.